src/test/ui/closures/2229_closure_analysis/diagnostics/liveness.rs

```diff
@@ -8,6 +8,7 @@
 struct Point {
     x: i32,
     y: i32,
+    //~^ WARNING: field is never read: `y`
```

Comment:
i think it still makes sense to add `allow(dead_code)` here as the test cases about something else